templates/release-semantic-automated-npm/release.config.ejs

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  "branches": <%- branches %>,
```

Comment:
Allowing branches to differ from a base setup adds a lot of complexity to the workflow. If a package needs a different branch setup, shouldn't it include a custom release.config.js instead?
```suggestion
  "branches": [
    "main",
    "next",
    {
      "name": "beta",
      "prerelease": true
    },
    {
      "name": "alpha",
      "prerelease": true
    }
  ],
```

Reply:
Done